crates/valence/examples/chat.rs

```diff
@@ -0,0 +1,90 @@
+use bevy_app::App;
+use tracing::warn;
+use valence::client::despawn_disconnected_clients;
+use valence::client::event::{default_event_handler, ChatCommand, ChatMessage};
+use valence::prelude::*;
+
+const SPAWN_Y: i32 = 64;
+
+pub fn main() {
+    tracing_subscriber::fmt().init();
+
+    App::new()
+        .add_plugin(ServerPlugin::new(()))
```

Comment:
We should default this example to offline mode to reduce the friction of testing with 2 clients.

Reply:
I think there is need in development config where offline mode is enabled by default, so all examples that use this config would be easier to play.

Reply:
Yeah, I edited this to the default for consistency with the rest of the examples. I do have to edit this myself too :)

For my actual project, I ended up doing this:

https://github.com/JoelEllis/plotsirv/blob/0d6695c55bc164cca3f1fb5c61a691b9c8a7ac56/src/main.rs#L12-L60

It'd be my preference to have this in offline mode too. If that's OK with @rj00a?

Reply:
Also, I had an idea of having an environment variable `VALENCE_MODE` that has possible values `production` and `development`, so in development environment offline mode would be enabled and things like encryption would be disabled by default.

Reply:
Yeah, that's a pretty good idea, but not in scope for this PR. Make an issue or a discord thread? :slightly_smiling_face:

Reply:
Ah, I prefer not to use Discord, but feel free to leave this idea there. Also, I need to think about this solution more thoroughly before submit the feature request.

Reply:
IMO you should create one as soon as you can - either place is good for letting other people throw their ideas into the ring and then come up with a good solution!

Reply:
> It'd be my preference to have this in offline mode too. If that's OK with @rj00a?

Yeah you can go ahead and change it.